homeassistant/components/vacuum/__init__.py

```diff
@@ -68,6 +68,7 @@
 STATE_PAUSED = STATE_PAUSED
 STATE_RETURNING = 'returning'
 STATE_ERROR = 'error'
+STATE_REMOTE_CONTROL = 'remote control'
```

Comment:
https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model

> If you want to add a new feature to an entity or any of its subtypes (light, switch, etc), you will need to propose it first in our architecture repo. Only additions will be considered that are common features among various vendors.